UnicornEngine/video/source/vulkan/Renderer.cpp

```diff
@@ -1049,6 +1099,23 @@ bool Renderer::CreateCommandPool()
     return true;
 }
 
+bool Renderer::CreateDepthBuffer()
+{
+    FreeDepthBuffer();
+
+    if (!FindDepthFormat(m_depthImageFormat))
+    {
+        LOG_ERROR("Not one of desired depth formats are not compatible!");
```

Comment:
Double negative. `Not one` ... `are not compatible` == `All are compatible`

`Device supporting desired depth format not found`?

Also do you need to actually do this check every time you call `CreateDepthBuffer` or would it suffice to call it only when `m_depthImageFormat` changes? If you can do it only on `m_depthImageFormat` change you could store the result in some boolean flag right after the value is changed